gui/webdisplay/src/TWebWindowsManager.cxx

```diff
@@ -707,7 +759,7 @@ bool ROOT::Experimental::TWebWindowsManager::Show(ROOT::Experimental::TWebWindow
 
    exec.ReplaceAll("$prog", prog.Data());
 
-   win.AddKey(key, where); // for now just application name
+   unsigned connid = win.AddProcId(batch_mode, key, where + rmdir); // for now just application name
 
    R__DEBUG_HERE("WebDisplay") << "Show web window in browser with:\n" << exec;
```

Comment:
"Showing"; "Show" can be misinterpreted as a command to the user (i.e. "please run this to see anything!")

Reply:
Yes, of course